### ISSUE TITLE
The obligatory naming discussion

### ISSUE THREAD

**Issue by kimdhamilton** [WebOfTrustInfo/btcr-hackathon-2019#10]
Let's kick it off here. Many of our names are outdated. What should they be now?

Naming discussion
- continuation/final/constructed DID Document
- implicit DID Document
- service/proof purpose
- audit trail generic vs specific (aka satoshi)

**Comment by ChristopherA** [comment]
We also need to discuss naming conventions for standard keys for BTCR-based DID Documents:

* #satoshi-transaction-key vs #satoshi
* #verifiable-credential-issuance-key #vckey-0

see example of usage in: https://github.com/WebOfTrustInfo/btcr-hackathon-2019/issues/4#issuecomment-519279631

-- Christopher Allen

**Comment by danpape** [comment]
Can I add something about txrefs here? We used to have "txref", then we talked about "txref-ext" or "txref-extended", which included our changes for adding the utxo index. Recently I have noticed people (including myself) say "short" and "long" txref. 

I suppose since now that we have had BIP-0136 accepted, there is no longer any need for "extended", so maybe "short txref" and "long txref" are OK? Or even, if we can help ourselves, just say "txref" all the time?

**Comment by peacekeeper** [comment]
@danpape :
> maybe "short txref" and "long txref" are OK?

Sounds good to me, just wanted to mention the [libtxref-java](https://github.com/dcdpr/libtxref-java) library calls it "extended", that should be made consistent then. BIP-0136 seems to call it "with outpoints".

**Comment by danpape** [comment]
Yes, I'll clean everything up in all the code once we settle on something.

As for "with outpoints" -- that is what the original BIP-0136 authors liked to use. Ryan and I looked into this and found that is how utxos and txos are referred to in the bitcoin code.

I'm hoping to issue a new PR to update BIP-0136 soon, once all of the implementations settle down as there are missing ones in the BIP.
